.github/workflows/crawl.yml

```diff
@@ -0,0 +1,71 @@
+name: Discv4 Crawl and DNS Update
+on:
+  schedule:
+    - cron: '0 */2 * * *'
+jobs:
+  build:
+    if: github.repository == 'ethereum/discv4-dns-lists'
+    name: Discv4-DNS-Crawler
+    runs-on: ubuntu-latest
+    env:
+      ETH_DNS_DISCV4_CRAWLTIME: 30m
+      ETH_DNS_DISCV4_PARENT_DOMAIN: ethdisco.net
+      ETH_DNS_DISCV4_KEY_PATH: ./secrets/dnskey.json
+      ETH_DNS_DISCV4_KEYPASS_PATH: ./secrets/dnskey_password.txt
+      CLOUDFLARE_API_TOKEN: ${{ secrets.CLOUDFLARE_API_TOKEN }}
+      ETH_DNS_CLOUDFLARE_ZONEID: ${{ secrets.ETH_DNS_CLOUDFLARE_ZONEID }}
+      ETH_DNS_DISCV4_KEY: ${{ secrets.ETH_DNS_DISCV4_KEY }}
+      ETH_DNS_DISCV4_KEYPASS: ${{ secrets.ETH_DNS_DISCV4_KEYPASS }}
+
+
+    steps:
+    - name: Set up Go
+      uses: actions/setup-go@v2-beta
+      with:
+        go-version: 1.13.8
+      id: go
+
+    - run: go version
+
+    - name: Check out code
+      uses: actions/checkout@v2
+
+    - name: Install dependencies
+      run: |
+        go get -u github.com/ethereum/go-ethereum/cmd/devp2p
+        go get -u github.com/ethereum/go-ethereum/cmd/ethkey
+
+    - name: Setup secrets
+      run: |
+        mkdir secrets
+        echo "$ETH_DNS_DISCV4_KEY" > "$ETH_DNS_DISCV4_KEY_PATH"
+        echo "$ETH_DNS_DISCV4_KEYPASS" > "$ETH_DNS_DISCV4_KEYPASS_PATH"
+
+    - name: Check env and secrets
+      run: |
+        ./.ci/deps.sh
+
+    - name: Crawl
+      run: |
+        ./.ci/crawl.sh
+
+    - name: Filter and sign
+      run: |
+        ./.ci/filter_and_sign.sh mainnet ropsten rinkeby goerli
+
+    - name: Commit and Push
+      env:
+        GITHUB_USER: FIXME
+        GITHUB_PAT: FIXME
+      run: |
+        git config --local user.name 'crawler'
+        git config --local user.email 'noreply@users.noreply.github.com'
+        git add all* les*
+        git commit --author 'crawler <>' -m "automatic update: crawl time ${ETH_DNS_DISCV4_CRAWLTIME} ci ${GITHUB_RUN_ID}:${GITHUB_RUN_NUMBER}"
+        git remote set-url origin https://${GITHUB_USER}:${GITHUB_PAT}@github.com/${GITHUB_REPOSITORY}.git
+        git push origin master
```

Comment:
Theoretically `actions/checkout@v2` should make it as easy as `git push`, but I just couldn't get that to work.